sqlstore/src/main/resources/db/migration/V8000__container_constraints.sql

```diff
@@ -0,0 +1,14 @@
+--StartNoTest
+ALTER TABLE Run_SequencerPartitionContainer ADD FOREIGN KEY (Run_runId) REFERENCES Run (runId);
+DELETE FROM Run_SequencerPartitionContainer where containers_containerId = 0;
+ALTER TABLE Run_SequencerPartitionContainer ADD FOREIGN KEY (containers_containerId) REFERENCES SequencerPartitionContainer (containerId);
+
+ALTER TABLE SequencerPartitionContainer_Partition ADD FOREIGN KEY (container_containerId) REFERENCES SequencerPartitionContainer (containerId);
+ALTER TABLE SequencerPartitionContainer_Partition ADD FOREIGN KEY (partitions_partitionId) REFERENCES _Partition (partitionId);
+
+ALTER TABLE SequencerPartitionContainer ADD UNIQUE (identificationBarcode);
```

Comment:
Won't this cause the migration to fail for EI since they have multiple nulls and several duplicates?